deployments/goerli/.chainId

```diff
@@ -0,0 +1 @@
+5
```

Comment:
what is this?

Reply:
those are artifacts from `hardhat-deploy` -- they duplicate what's in `.openzeppelin` so I'll gitignore and remove them.